region/maridia/inner-pink.json

```diff
@@ -2393,6 +1542,10 @@
                     {"or": [
                       "canGravityJump",
                       "canFly",
+                      {"canComeInCharged": {
+                        "fromNode": 1,
+                        "framesRemaining": 50
+                      }},
```

Comment:
No directlink on this one?

Reply:
It was intentional, because the two nodes might just as well overlap. I don't see a reason for it not to work, in fact I added `fromNode` because I intended not to have direct links and instead use the same `canComeInCharged` for several linear links. Then I realized that doesn't work if you're visiting several nodes because you can't really stop the spark at will.

But here, I think it should be fine.